control-plane/connect-inject/peering_acceptor_controller.go

```diff
@@ -52,16 +54,32 @@ func (r *PeeringAcceptorController) Reconcile(ctx context.Context, req ctrl.Requ
 	// If the PeeringAcceptor resource has been deleted (and we get an IsNotFound
 	// error), we need to delete it in Consul.
 	if k8serrors.IsNotFound(err) {
-		r.Log.Info("PeeringAcceptor was deleted, deleting from Consul", "name", req.Name, "ns", req.Namespace)
-		if err := r.deletePeering(ctx, req.Name); err != nil {
-			return ctrl.Result{}, err
-		}
+		r.Log.Info("PeeringAcceptor resource not found. Ignoring.", "name", req.Name, "ns", req.Namespace)
```

Comment:
We do not need to worry about the not found error here as this would imply the CRD was never reconciled. If it was, it would have a finalizer so it is guaranteed to go through the workflow that has just been added.